protocols/rendezvous/src/server.rs

```diff
@@ -224,6 +224,7 @@ impl NetworkBehaviour for Behaviour {
 
                         return Poll::Ready(new_to_swarm);
                     }
+                    _ => {}
```

Comment:
@thomaseizinger is this a bug as well? Should it return Poll::Pending in this case?

Reply:
It is yes, we should break out of the loop in case `self.inner.poll` returns `Poll::Pending`.

Reply:
this is not a bug because it just falls through to the `return Poll::Pending` a the bottom of the function.

Reply:
Right, thanks for digging into it!